src/main/java/eu/dissco/core/translator/domain/DigitalMedia.java

```diff
@@ -2,16 +2,15 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.databind.JsonNode;
-import eu.dissco.core.translator.schema.DigitalEntity;
-import eu.dissco.core.translator.terms.specimen.PhysicalSpecimenId;
+import eu.dissco.core.translator.terms.specimen.PhysicalSpecimenID;
 
-public record DigitalMediaObject(
+public record DigitalMedia(
```

Comment:
Maybe call this DigitalMediaWrapper to be aligned with DigitalSpecimenWrapper?

Reply:
:+1: